src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePluginTest.java

```diff
@@ -888,7 +568,7 @@ public void testGetInstanceSuccessfulWithoutVolatileDiskResource() throws Fogbow
 		PowerMockito.verifyStatic(OpenNebulaClientUtil.class, VerificationModeFactory.times(1));
 		OpenNebulaClientUtil.getVirtualMachine(Mockito.eq(client), Mockito.anyString());
 
-		Mockito.verify(this.plugin, Mockito.times(1)).doComputeInstance(virtualMachine);
+		Mockito.verify(this.plugin, Mockito.times(1)).doGetInstance(virtualMachine);
```

Comment:
The same previous suggestion for this verify.

Reply:
done